src/tools/PlaywrightSharp.Tooling/IdentifyMissingTests.cs

```diff
@@ -65,27 +65,28 @@ public static void Run(IdentifyMissingTestsOptions options)
             int noMatches = 0;
             int totalTests = 0;
 
-            List<(string FileName, string TestName)> missingTests = new();
-            List<KeyValuePair<(string FileName, string TestName), List<(string FileName, string TestName)>>> invalidMaps = new();
-            foreach (var atx in attributes)
+            List<PlaywrightTestAttribute> missingTests = new();
+            List<KeyValuePair<PlaywrightTestAttribute, List<PlaywrightTestAttribute>>> invalidMaps = new();
+            foreach (var x in _testPairs)
             {
                 totalTests++;
 
                 // a test can either be a full match, a partial (i.e. just the test name) or no match
-                var potentialMatch = _testPairs.Where(x => string.Equals(x.TestName, atx.TestName, StringComparison.InvariantCultureIgnoreCase));
+                var potentialMatch = attributes.Where(atx => string.Equals(x.TestName, atx.TestName, StringComparison.InvariantCultureIgnoreCase))
+                                               .Where(atx => string.Equals(x.Describe, atx.Describe, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
@avodovnik Without this additional `.Where` term:
`Found/Mismatched/Missing: 695/585/515 out of 1795`

Reply:
The other 8 are due to one upstream test being attributed to multiple `PlaywrightTest` attributes.

Reply:
We don't need two wheres here. It could be only one with an `&&`

Reply:
I think it would be great if you can find one example so we can check what's right or wrong.